packages/core/admin/server/policies/isAuthenticatedAdmin.js

```diff
@@ -1,9 +1,5 @@
 'use strict';
 
-module.exports = (ctx, next) => {
-  if (!ctx.state.isAuthenticated) {
-    return ctx.unauthorized();
-  }
-
-  return next();
+module.exports = ({ ctx }) => {
+  return ctx.state.isAuthenticated;
```

Comment:
I'm nipicking here :D
```suggestion
  return Boolean(ctx.state.isAuthenticated);
```